core/src/worker/workflow/wft_poller.rs

```diff
@@ -74,6 +77,8 @@ pub(crate) struct WFTPollerShared {
     last_seen_sticky_backlog: (watch::Receiver<usize>, watch::Sender<usize>),
     sticky_active: OnceLock<watch::Receiver<usize>>,
     non_sticky_active: OnceLock<watch::Receiver<usize>>,
+    wait_for_first_nonsticky_poll: Notify,
```

Comment:
Neither of these two new fields are ever changed/notified. Uncompleted work on an optimization?